apiserver/tools.go

```diff
@@ -91,12 +92,12 @@ func (h *toolsDownloadHandler) processGet(r *http.Request, st *state.State) ([]b
 		return nil, errors.Annotate(err, "error getting tools storage")
 	}
 	defer storage.Close()
-	_, reader, err := storage.Tools(version)
+	_, reader, err := storage.Open(v)
```

Comment:
Or perhaps go with the original code and use version.String() so that we know it's in canonical form.